app/views/Board.coffee

```diff
@@ -0,0 +1,73 @@
+Cell = require './Cell'
+
+class Board
+
+  constructor: (@grid, @two) ->
+
+    @size = @two.height * .80
+    offset = @size * .025
+    width = ( @size - offset ) / ( @grid.dimension ) - offset
+
+    @x = ( @two.width ) / 2
+    @y = ( @two.height ) / 2
+    @y = if @y < @size / 2 then @size / 2 else @y
+
+    board = @two.makeRectangle @x, @y, @size, @size
+    board.noStroke().fill =  '#c2d1e6' #'#B0C4DE' #'#F0F8FF' #'#CDE1FF'
+    board.visible = true
+
+    @change = offset + width
+
+    cellWidth = ((@size - offset ) / @grid.dimension ) - offset
+
+    @createEmptyCells cellWidth - 5
+
+    @createCells cellWidth
+
+  createEmptyCells: (width) ->
+    @cells = []
+    for row in [0...@grid.dimension]
+      @cells.push []
+      for col in [0...@grid.dimension]
+        cell = new Cell col, row, width, @two, @
+        cell.setColor '#b7c9e1'
+        cell.setBorder '#b7c9e1'
+        @cells[row].push cell
+
+  createCells: (width) ->
+    @cells = []
+    for row in [0...@grid.dimension]
+      @cells.push []
+      for col in [0...@grid.dimension]
+        cell = new Cell col, row, width, @two, @
```

Comment:
Change the last parameter to ```this``` for clarity. Same on line 32. ```@``` should only be used whenever there's a variable after